src/react-app-env.d.ts

```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```

Comment:
@davelosert  Either linter or prettier added this file. Do you know what it is and if we need it?

Reply:
@sincovschi : I think it was `react-scripts` which added it, though I do not know why.

As we now only use react-scripts for testing (and not for the example anymore), we could probably also delete it and add the test dependencies (basically just `jest`) and configuration directly in our project. That would again make the whole thing smaller. 

WDYT? I can draw up a PR to do this if you like.

Reply:
Haha, WDYT means "What do you think?".
Will commit to this branch. :)

Reply:
@sincovschi : Done, do you want to have a look?